bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,230 @@ class StylintBear:
+            names. If ``class_naming_convention`` is set to ``true``,
+            ``variable_naming_convention`` will  also look at class and
+            id names.
+        :param none_keyword:
+            If ``True`` check for places where ``none`` used instead of 0. If
+            ``False`` check for places where 0 could be used instead of none.
+        :param check_no_important_keyword:
+            If ``true``, show warning when ``!important`` is found.
+        :param spaces_inside_parentheses:
+            Enforce or disallow use of extra spaces inside parentheses.
+        :param placeholder:
+            Enforce extending placeholder vars when using ``@extend(s)``.
+        :param prefix_vars_with_dollar:
+            Enforce use of ``$`` when defining a variable. In Stylus using a
+            ``$`` when defining a variable is optional, but is a good idea
+            if you want to prevent ambiguity. Not including the ``$`` sets up
+            situations where you wonder: "Is this a variable or a value?"
+            For instance: padding ``$default`` is easier to understand than
+            padding ``default``.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param semicolons:
+            When ``True``, allow semicolons. When ``False``, disallow
+            semicolons. When ``None``, do not check either way.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param sort_order:
+            Enforce a particular sort order when declaring properties. Throws
+            a warning if you don't follow the order. If set to ``None``, allow
+            any order.
+            Example if ``alphabetical``:
+            prefer this:
+            ```
+            .some-class
+                display block
+                float left
+                position absolute
+                right 10px
+                top 0
+            ```
+            over this:
+            ```
+            .some-class
+                position absolute
+                top 0
+                right 10px
+                display block
+                float left
+            ```
+        :param stacked_properties:
+            No one-liners. Allow putting properties on new lines.
+            Example if ``False``: prefer
+            ```
+            .className
+                padding 0
+            ```
+            over
+            ``.className { padding 0 }``
+        :param check_valid_property:
+            Check that a property is valid CSS or HTML.
+        :param zero_units:
+            Looks for instances of ``0px``. You don't need the ``px``.
+            Checks all units, not just ``px``.
+        :param z_index_normalize:
+            Enforce some basic z-index sanity. Any number passed in
+            will be used as the base for your z-index values. Throws an error
+            if your value is not normalized. Use ``0`` to disable this feature.
```

Comment:
Normalized means a multiple of the number that you pass in. So, if you pass in 5, it will prefer multiples of 5 like 10, 15, 20, etc.